src/main/webapp/app/exercises/modeling/participate/modeling-submission.service.ts

```diff
@@ -52,10 +52,11 @@ export class ModelingSubmissionService {
      * @param {number} exerciseId - Id of the exercise
      * @param {any?} req - Request option
      */
-    getModelingSubmissionsForExercise(exerciseId: number, req?: any): Observable<HttpResponse<ModelingSubmission[]>> {
+    getModelingSubmissionsForExerciseByCorrectionRound(exerciseId: number, req?: any, correctionRound?: number): Observable<HttpResponse<ModelingSubmission[]>> {
+        correctionRound = correctionRound ?? 0;
```

Comment:
You can simplify it by using a default value of 0.

```suggestion
    getModelingSubmissionsForExerciseByCorrectionRound(exerciseId: number, req?: any, correctionRound = 0): Observable<HttpResponse<ModelingSubmission[]>> {
```

Reply:
thanks!